verilog/tools/ls/autoexpand_test.cc

```diff
@@ -163,118 +198,715 @@ module t(/*AUTOARG*/
 endmodule)");
 }
 
-TEST(Autoexpand, SkipPredeclared) {
+TEST(Autoexpand, AUTOARG_SkipPredeclared) {
   TestTextEdits(GenerateAutoExpandTextEdits,
                 R"(
-module t(i,
+module t(input i1, i2,
          o1, /*AUTOARG*/
 //Inputs
 clk, rst
 );
   input logic clk;
   input logic rst;
-  input logic i;
+  input logic i2;
   output logic o1;
   output logic o2;
 endmodule)",
                 R"(
-module t(i,
+module t(input i1, i2,
          o1, /*AUTOARG*/
   // Inputs
-  clk,
-  rst,
+  clk, rst,
   // Outputs
   o2
   );
   input logic clk;
   input logic rst;
-  input logic i;
+  input logic i2;
   output logic o1;
   output logic o2;
 endmodule)");
 }
 
+TEST(Autoexpand, AUTOINST_ExpandEmpty) {
+  TestTextEdits(GenerateAutoExpandTextEdits,
+                R"(
+module bar(input i1, output o1);
```

Comment:
What happens if we have two conflicting versions of 'bar' , say one with two and one with three parameters. What happens ? (a) see the conflict and refuse to AUTOINST, (b) take the first or last known one (c) crash ?
Might be worthwhile a test.
I think (a) would be most desirable, though (b) is probably an initially acceptable solution as well.

Reply:
Added a test. (b) is what happens, due to `SymbolTableHandler` only being able to find one definition for now.

I think we can revisit this once we're able to return multiple locations for `textDocument/definition`. We could use LSP's `window/showMessage` to display an error for the user.